spinalcordtoolbox/deepseg/models.py

```diff
@@ -342,6 +351,15 @@
                              'the Canadian Prospective Cohort Study (CanProCo).',
          'url': 'https://github.com/ivadomed/canproco',
          'models': ['model_seg_sc_lesion_canproco_nnunet']},
+    'seg_sc_epi':
+        {'description': 'Spinal cord segmentation for EPI data',
```

Comment:
I'm not 100% sure about the terminology, but I notice that the contrast is `BOLD`, and the repo also mentions "fMRI". I also see the term "EPI-BOLD fMRI" in some papers like [this](https://pubmed.ncbi.nlm.nih.gov/14696216/).

So, maybe it's also worth mentioning this in the task name and short description?

```suggestion
    'seg_sc_epi-bold_fmri':
        {'description': 'Spinal cord segmentation for EPI-BOLD fMRI data',
```

Reply:
I'm not a huge fan of "-" in task name, because the way i install them is:
- run help
- identify the task
- double click on task and copy
- paste in syntax with flag `-install`

with the "-" the double click will faile

Reply:
the comment about the BOLD contrast if valid-- however in this case, i don't think we should alter the task name

moreover, the distinction is really about gre (gradient echo, used for fMRI) vs. spin echo EPI (used for diffusion)-- 

this comment can go in the long description

Reply:
Thank you for the feedback! Very good point about `-`, and I appreciate the insight re: dMRI/fMRI terminology.

Reply:
Reverted now